flotta-agent.spec

```diff
@@ -77,8 +81,13 @@ make install-worker-config USER=%{flotta_user} HOME=/home/%{flotta_user} LIBEXEC
 
 %post race
 loginctl enable-linger %{flotta_user}
-systemctl start user@$(getent passwd %{flotta_user} | cut -d: -f3).service
```

Comment:
Actually, we need to keep this. It starts the systemd for the user flotta, without it we can't start the podman.socket for the user.

Reply:
So I just tested, and it seems, the podman.socket start, start the systemd service for the user, it seems the issues I had previously was because of the selinux. So it seems it's not needed anymore.